chai-events.js

```diff
@@ -95,7 +95,7 @@ function plugin(chai, utils) {
           if(done) { return; }
           done = true;
           try {
-            assert(false, "expected #{this} to emit "+name.toString()+".");
+            assert(false, '', "expected #{this} to emit "+name.toString()+".");
```

Comment:
No need this changes here. Previous assert was corect. 
First assert() message is about 'positive' case and second for 'negative' (flag)

Reply:
Hmm, looks like you're right.  The `AssertionError` didn't seem to have the right message without the extra quotes, but I can't figure out why.  Looks like I'll revert it!